backend/coins/eth/account.go

```diff
@@ -735,9 +738,10 @@ func (account *Account) FeeTargets() ([]accounts.FeeTarget, accounts.FeeTargetCo
 	return feeTargets, accounts.DefaultFeeTarget
 }
 
-// gasPrice returns the currently suggested gas price for the given fee target, or a custom gas
-// price if the fee target is `FeeTargetCodeCustom`.
-func (account *Account) gasPrice(args *accounts.TxProposalArgs) (*big.Int, *big.Int, error) {
+// gasFees returns the currently suggested maxFeePerGas and maxPriorityFee for the given fee target, or a custom fee
+// if the fee target is `FeeTargetCodeCustom`. The custom fee sets both maxFeePerGas and maxPriorityFee to the same value.
+// TODO: The UI should have and advanced setting to allow the user to set maxFeePerGas and maxPriorityFee separately.
```

Comment:
what do Rabby and MEW do with custom fees? Two separate fields? sounds like very difficult UX.

Reply:
From what I saw, MEW doesn't let you set a custom fee at all, and Rabby does the same as this PR currently does, just one custom fee field which sets both maxFeePerGas and maxPriorityFeePerGas to that same custom fee value. I guess it's only MetaMask really from the most commonly used wallets that lets you set the fees separately in advanced mode.

I'll delete the TODO and we can revisit this if there's ever demand for separate custom fee fields